server/src/main/kotlin/org/elaastic/player/ComponentTestingController.kt

```diff
@@ -613,21 +612,16 @@ class ComponentTestingController(
 
         model.addAttribute("user", user)
         model.addAttribute(
-            "statementPanelModel",
-            StatementPanelModel(
+            "statementInfoPanelModel",
+            StatementInfoPanelModel(
+                title = "Énoncé de test",
```

Comment:
Because this is test content, I agree it not necessary to use i18n here, but I would prefer English at least.